spring-integration-file/src/main/java/org/springframework/integration/file/filters/ChainFileListFilter.java

```diff
@@ -31,11 +32,25 @@
  *
  * @author Artem Bilan
  * @author Gary Russell
+ * @author Cengis KOCAURLU
  *
  * @since 4.3.7
  *
  */
 public class ChainFileListFilter<F> extends CompositeFileListFilter<F> {
+	/**
+	* {@inheritDoc}
+	*/
+	public ChainFileListFilter() {
+		super();
+	}
+
+	/**
+	* {@inheritDoc}
+	*/
+	public ChainFileListFilter(Collection<? extends FileListFilter<F>> fileFilters) {
```

Comment:
May we have some simple test-case with the mentioned in the issue XML config?

Reply:
i updated the `ChainFileListFilterIntegrationTests`file